.github/workflows/release_linux_pytorch_wheels.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["cp311-cp311", "cp312-cp312", cp313-cp313]
```

Comment:
Let's set a name on this job that includes "test". Right now the name is derived from

```yml
jobs:
  build:
    strategy:
      fail-fast: false
      matrix:
        python_version: ["3.11", "3.12", 3.13]
```

so "build (3.11)", etc. when I'd prefer more like "Build/test (3.11, gfx942)"

See https://github.com/ROCm/TheRock/actions/runs/16201610480

Reply:
Added a better name.